provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -106,322 +111,322 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 				},
 			},
-			"name": tfrschema.StringAttribute{
+			"name": schema.StringAttribute{
 				Description: "Name of the cluster. Cannot exceed 15 characters in length.",
 				Required:    true,
 				PlanModifiers: []planmodifier.String{
-					ValueCannotBeChangedModifier(),
+					stringplanmodifier.RequiresReplace(),
 				},
 			},
-			"cloud_region": tfrschema.StringAttribute{
+			"cloud_region": schema.StringAttribute{
 				Description: "Cloud region identifier, for example 'us-east-1'.",
 				Required:    true,
 			},
-			"sts": tfrschema.SingleNestedAttribute{
+			"sts": schema.SingleNestedAttribute{
 				Description: "STS configuration.",
 				Attributes:  stsResource(),
 				Optional:    true,
 			},
-			"multi_az": tfrschema.BoolAttribute{
+			"multi_az": schema.BoolAttribute{
 				Description: "Indicates if the cluster should be deployed to " +
-					"multiple availability zones. Default value is 'false'.",
+					"multiple availability zones. Default ValueString() is 'false'.",
 				Optional: true,
 				Computed: true,
 				PlanModifiers: []planmodifier.Bool{
 					boolplanmodifier.UseStateForUnknown(),
-					ValueCannotBeChangedModifier(),
+					boolplanmodifier.RequiresReplace(),
 				},
 			},
-			"disable_workload_monitoring": tfrschema.BoolAttribute{
+			"disable_workload_monitoring": schema.BoolAttribute{
 				Description: "Enables you to monitor your own projects in isolation from Red Hat " +
 					"Site Reliability Engineer (SRE) platform metrics.",
 				Optional: true,
 			},
-			"disable_scp_checks": tfrschema.BoolAttribute{
+			"disable_scp_checks": schema.BoolAttribute{
 				Description: "Enables you to monitor your own projects in isolation from Red Hat " +
 					"Site Reliability Engineer (SRE) platform metrics.",
 				Optional: true,
 				PlanModifiers: []planmodifier.Bool{
-					ValueCannotBeChangedModifier(),
+					boolplanmodifier.RequiresReplace(),
 				},
 			},
-			"properties": tfrschema.MapAttribute{
+			"properties": schema.MapAttribute{
 				Description: "User defined properties.",
 				ElementType: types.StringType,
 				Optional:    true,
 				Computed:    true,
-				Validators:  propertiesValidators(),
+				Validators:  []validator.Map{PropertiesValidator()},
 			},
-			"ocm_properties": tfrschema.MapAttribute{
+			"ocm_properties": schema.MapAttribute{
 				Description: "Merged properties defined by OCM and the user defined 'properties'.",
 				ElementType: types.StringType,
 				Computed:    true,
 			},
-			"tags": tfrschema.MapAttribute{
-				Description: "Apply user defined tags to all resources created in AWS.",
-				ElementType: types.StringType,
-				Optional:    true,
-				PlanModifiers: []planmodifier.Map{
-					ValueCannotBeChangedModifier(),
-				},
+			"tags": schema.MapAttribute{
+				Description:   "Apply user defined tags to all resources created in AWS.",
+				ElementType:   types.StringType,
+				Optional:      true,
+				PlanModifiers: []planmodifier.Map{},
```

Comment:
maybe you miss stringplanmodifier.RequiresReplace() ?

Reply:
missed it thanks